openslides/mediafiles/serializers.py

```diff
@@ -1,14 +1,38 @@
-from openslides.utils.rest_api import ModelSerializer, SerializerMethodField
+import mimetypes
 
+from django.db import models as dbmodels
+from rest_framework.fields import FileField
```

Comment:
For unknown reason we import all rest_framework stuff from our utils.rest_api so we can change this globally later if required. So you have to add it there and change the import. Maybe use `# noqa` to disabled flake8 over there.